### DISCUSSION TITLE
Automatically install Playwright browsers

### DISCUSSION THREAD

**Discussion by andrecasal** [epicweb-dev/epic-stack#93] [Ideas]
I wanted to know why my Playwright tests were not running locally and then realized I had no browsers installed.

Wouldn't we want the browsers installed from the start?

**Comment by kentcdodds** [comment]
That could definitely be something we add as part of the setup script. Would you be interested in doing that?

**Reply by kentcdodds** [reply]
I've opened an issue for this: https://github.com/epicweb-dev/epic-stack/issues/94

**Reply by andrecasal** [reply]
I'd be happy to 👍
